content/x-ray/x-ray.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "X-Ray Console"
+date: 2018-11-177T08:30:11-07:00
+weight: 13
+draft: false
+---
+
+We now have the example microservices deployed, so we are going to investigate our [Service Graph](https://docs.aws.amazon.com/xray/latest/devguide/xray-concepts.html#xray-concepts-servicegraph) and [Traces](https://docs.aws.amazon.com/xray/latest/devguide/xray-concepts.html#xray-concepts-traces) in [X-Ray section of the AWS Management Console](https://us-west-2.console.aws.amazon.com/xray/home?region=us-west-2#/service-map).
+
+The [Service map in the console](https://us-west-2.console.aws.amazon.com/xray/home?region=us-west-2#/service-map) provides a visual representation of the steps identified by X-Ray for a particular trace. Each resource that sends data to X-Ray within the same context appears as a service in the graph.
+
```

Comment:
can you add some details into what we are seeing in this picture? For ex, in this picture, we have two services running on EKS ....and we can see frontend is able to do 39 transactions per sec with average of 0.99 msec latency